pkg/bootstrapper/bootstrapper.go

```diff
@@ -49,6 +49,13 @@ const (
 	certDirectory = "c:\\var\\lib\\kubelet\\pki\\"
```

Comment:
Are we still going with this approach of fixing bug while rewriting the way we are handling kubelet args? I am still a nack on this. To be clear, I think we need to do this but not when we are fixing a bug.

Reply:
I dont see how its possible to not change the way we are handling kubelet args. The previous way does not work when the value of the arg is no longer just a word. I've removed the change to strict string parsing and gone back to using regex's to avoid making things too different than before.